classes/statistics/TemporaryItemRequestsDAO.php

```diff
@@ -30,7 +30,10 @@ class TemporaryItemRequestsDAO extends PKPTemporaryItemRequestsDAO
      */
     protected function getInsertData(object $entryData): array
     {
-        return ['chapter_id' => $entryData->chapterId];
+        return array_merge(
+            parent::getInsertData($entryData),
+            ['chapter_id' => $entryData->chapterId]
+        );
```

Comment:
Not important, but as it's a dictionary, it's possible to replace by `return parent::getInsertData($entryData) + ['chapter_id' => $entryData->chapterId];`